tools/stern/test/integration/API.hs

```diff
@@ -1,7 +1,8 @@
 {-# LANGUAGE OverloadedRecordDot #-}
+{-# OPTIONS_GHC -Wno-ambiguous-fields #-}
+{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
afaiu you didn't add any orphans and the other warning doesn't seem like we want to keep it, do we? 
```suggestion
```

Reply:
There is an orphan instance there.